src/activity_handlers.cpp

```diff
@@ -330,12 +332,14 @@ static void butcher_cbm_group( const std::string &group, const tripoint &pos,
     if( check_butcher_cbm( roll ) ) {
         //The CBM works
         const auto spawned = g->m.put_items_from_loc( group, pos, age );
-        for( const auto &it : spawned ) {
+        for( item *it : spawned ) {
             add_msg( m_good, _( "You discover a %s!" ), it->tname() );
+            it->set_flag( "FILTHY" );
```

Comment:
Would probably be better to move this *before* the message. Currently the message will announce a clean item (no FILTHY tag on it). But the player will actually find a filthy item on the ground.